### PR TITLE
Videos array and buttons outline

### PR DESCRIPTION
**Pull request by lalitmee**
### Bugs Fixed

1. Videos array was missing because of a single condition fail.
2. I also removed the outline from the button when we click on them, it was looking odd.

### CODE REVIEW

**Comment by aashutoshrathi** [comment]
I checked it and it works for the first video as expected but not for further.

**Comment by lalitmee** [comment]
Yeah as I told you that on scrolling a lot of **images** array or **videos** array creating some problem or we are missing something. :thinking:

**Comment by aashutoshrathi** [comment]
I think I should we should check it once more before merging.
I'll ping you after tweaking some more

**Comment by lalitmee** [comment]
See just now I went to Instagram and its working for the first 3 videos.

peek 2018-12-25 17-08

File: download.js [hunk 1]
```diff
@@ -73,9 +78,8 @@ function injectButtons () {
 
   if (videos[0]) {
     for (i = 0; i < videos.length; i += 1) {
-      if (!videos[i].src.endsWith('&dl=1')) {
-        videos[i].src = `${videos[i].src}&dl=1`
-      }
+      videos[i].poster = `${videos[i].poster}&dl=1`
+      videos[i].src = `${videos[i].currentSrc}&dl=1`
```
Comment (aashutoshrathi) [inline download.js]: Make it
```js
if (videos[0]) {
    for (i = 0; i < videos.length; i += 1) {
      if (!videos[i].src.endsWith('&dl=1')) {
        videos[i].src = `${videos[i].currentSrc}&dl=1`
      }
      if (!videos[i].poster.endsWith('&dl=1')) {
        videos[i].poster = `${videos[i].poster}&dl=1`
      }
    }
  }
```

**Comment by aashutoshrathi** [comment]
Also, once we get this error

screenshot from 2018-12-25 18-12-18

Things start screwing up

**Pull request merged**